packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -64,7 +65,8 @@ class InspectorController implements InspectorServiceClient {
             InspectorObjectGroupManager(inspectorService, 'selection') {
     _refreshRateLimiter = RateLimiter(refreshFramesPerSecond, refresh);
 
-    inspectorTree = inspectorTreeFactory(
+    assert(inspectorTree != null);
```

Comment:
if it is required do we need this assert?

Reply:
yes. the assert is the runtime companion to the `@required` static check.